backend/src/polling_station/structs.rs

```diff
@@ -316,33 +423,205 @@ impl Validate for VotesCounts {
+    #[schema(value_type = u32)]
+    pub total_admitted_voters_recount: Count,
+}
+
+/// Check if all voters recounts and votes counts are equal to zero.
+/// Used in validations where this is an edge case that needs to be handled.
+fn all_zero_voters_recounts_and_votes_counts(voters: &VotersRecounts, votes: &VotesCounts) -> bool {
+    voters.poll_card_recount == 0
+        && voters.proxy_certificate_recount == 0
+        && voters.voter_card_recount == 0
+        && voters.total_admitted_voters_recount == 0
+        && votes.votes_candidates_counts == 0
+        && votes.blank_votes_count == 0
+        && votes.invalid_votes_count == 0
+        && votes.total_votes_cast_count == 0
+}
+
+/// Check if the voters recounts and votes counts are identical, which should result in a warning.
+///
+/// This is not implemented as Eq because there is no true equality relation
+/// between these two sets of numbers.
+fn identical_voters_recounts_and_votes_counts(
+    voters: &VotersRecounts,
+    votes: &VotesCounts,
+) -> bool {
+    !all_zero_voters_recounts_and_votes_counts(voters, votes)
+        && voters.poll_card_recount == votes.votes_candidates_counts
+        && voters.proxy_certificate_recount == votes.blank_votes_count
+        && voters.voter_card_recount == votes.invalid_votes_count
+        && voters.total_admitted_voters_recount == votes.total_votes_cast_count
+}
+
+impl Validate for VotersRecounts {
+    fn validate(
+        &self,
+        election: &Election,
+        validation_results: &mut ValidationResults,
+        field_name: String,
+    ) {
+        // validate all counts
+        self.poll_card_recount.validate(
+            election,
+            validation_results,
+            format!("{field_name}.poll_card_recount"),
+        );
+        self.proxy_certificate_recount.validate(
+            election,
+            validation_results,
+            format!("{field_name}.proxy_certificate_recount"),
+        );
+        self.voter_card_recount.validate(
+            election,
+            validation_results,
+            format!("{field_name}.voter_card_recount"),
+        );
+        self.total_admitted_voters_recount.validate(
+            election,
+            validation_results,
+            format!("{field_name}.total_admitted_voters_recount"),
+        );
+
+        // F.13 validate that total_admitted_voters_recount == poll_card_recount + proxy_certificate_recount + voter_card_recount
+        if self.poll_card_recount + self.proxy_certificate_recount + self.voter_card_recount
+            != self.total_admitted_voters_recount
+        {
+            validation_results.errors.push(ValidationResult {
+                fields: vec![
+                    format!("{field_name}.total_admitted_voters_recount"),
+                    format!("{field_name}.poll_card_recount"),
+                    format!("{field_name}.proxy_certificate_recount"),
+                    format!("{field_name}.voter_card_recount"),
```

Comment:
These say `votes_recounts` instead of `voters_recounts` in the response, causing the error not to show up.

Reply:
Fixed in e6bda22191ad310081f5d8140ea216514599e40c. Thanks again for the extensive testing